beesdoo_easy_my_coop/__manifest__.py

```diff
@@ -24,6 +24,7 @@
 
     'data': [
         'views/res_company.xml',
+        'views/res_partner.xml',
```

Comment:
remove since the file doesn't exist anymore

Reply:
i put the xml file